README.md

```diff
@@ -80,6 +80,14 @@ some of the complexity calculations:
   Skips creating the visibility matrix and calculating the coreSize,
   which can be very expensive for large projects
 
+#### parserOptions
+
+Espree is used to parse the JavaScript AST.
+By default, it uses the options `{ ecmaVersion: 7, loc: true }`.
```

Comment:
After merging your work in #205, I believe the default `ecmaVersion` config property is `8` now. :)